images/docker/cbdb/build/rocky9/Dockerfile

```diff
@@ -138,7 +138,7 @@ RUN dnf makecache && \
         libyaml-devel \
         perl-IPC-Run \
         protobuf-devel && \
-    dnf clean all && \
+    dnf clean all && rm -rf /var/cache/dnf/* && \
```

Comment:
Maybe amend the commit message with the following which describes this change.

Purpose of rm -rf /var/cache/dnf/*

Even though dnf clean all should remove the contents of /var/cache/dnf/, this extra rm step is:

* A failsafe/manual cleanup, in case DNF leaves anything behind (e.g. due to bugs or distro version differences).
* Common in slim or hardened images to ensure absolutely no cache data remains.

Reply:
Good. Let me leave it in the next single PR.